pkg/scheduler/cache/cache.go

```diff
@@ -258,6 +260,19 @@ func newSchedulerCache(config *rest.Config, schedulerName string, defaultQueue s
 		panic(fmt.Sprintf("failed init eventClient, with err: %v", err))
 	}
 
+	// create default queue
+	defaultQue := v1alpha2.Queue{
+		ObjectMeta: metav1.ObjectMeta{
+			Name: defaultQueue,
+		},
+		Spec: v1alpha2.QueueSpec{
+			Weight: 1,
+		},
+	}
+	if _, err := kbClient.SchedulingV1alpha2().Queues().Create(&defaultQue); err != nil && !apierrors.IsAlreadyExists(err) {
```

Comment:
What if I have a existing default-queue from the previous version of volcano?

Reply:
Does not matter